src/test/datascience/jupyter/kernels/installationPrompts.vscode.test.ts

```diff
@@ -72,11 +71,7 @@ suite('DataScience Install IPyKernel (slow) (install)', function () {
         if (IS_REMOTE_NATIVE_TEST) {
             return this.skip();
         }
-        if (
-            (IS_CI_SERVER && getOSType() !== OSType.OSX) ||
```

Comment:
We don't have tests on CI running on a Mac, probably meant `if not CI and os is Mac then skip`

Reply:
Will this enable them for Windows, or does that have its own filters?

Reply:
Enabled for all